.jshintignore

```diff
@@ -0,0 +1,2 @@
+js/skip-link-focus-fix.js
+**.min.js
```

Comment:
> Why are you excluding skip-link-focus-fix.js?

See above.

> I do not think that **.min.js is necessary here.

Although `_s` does not supply minified files (may be it should), themes based on `_s` often will. Having a sane base for `.jshintignore` made sense when I set it up.